### PR TITLE
[#1194] Maintain Website / GDPR / Rules / etc

### PR DESCRIPTION
**Pull request by max-leuthaeuser**
- do we fullfill the header footer whatever stuff the eclipse fqn wants to have? yes, looks good. Exactly like on the eclipse.org main page.
 - do we need tagmanager and analytics? tagmanager JS stuff has thrown exceptions - I removed it for now, may be reverted if needed
 - do we need embedded twitter? No really, as it was only used to display inline tweets - I removed all twitter stuff and added simple links to @xtext and @xtendlang instead
 - do we need this inform user on cookies thingy? I added a Cookie Consent thing using cookieconsent.insites.com; TODO: what to do if the users does not want to opt-in? See TODOs in e.g., xtext-website/_includes/head.html
 - can the fork me on github thing done without pulling third party stuff directly? I removed the link to the external IMG source and used a local one instead. (xtext-website/images/forkme_right_white_ffffff.png)

### CODE REVIEW

File: xtext-website/_includes/footer.html [hunk 1]
```diff
@@ -19,22 +19,13 @@ <h3 class="footer-links-header">Quick Links</h3>
...
-                fjs.parentNode.insertBefore(js,fjs);
-              }
-            }(document,"script","twitter-wjs");
-            </script>
-
+            <!-- Social Media Links -->
+            <h3 class="footer-links-header"">Social Media</h3>
+            <ul class="footer-links clearfix">
+              <li>
+                <a href="https://twitter.com/xtext"><img src="{{ site.baseurl }}/images/Twitter-bird-darkgray.png" class="img-responsive" style="margin-right: 5px;height: 1em;" alt="Twitter icon">@xtent on Twitter</a>
```
Comment (cdietrich) [inline xtext-website/_includes/footer.html]: typo: Xtext on Twitter

**Comment by cdietrich** [comment]
xtend
icon missing?

**Comment by cdietrich** [comment]
it looks like the both now behave differently regarding analytics. do i overlook someth

**Comment by max-leuthaeuser** [comment]
I did not change anything w.r.t. Google analytics, only removed the tagmanager thing.

**Comment by cdietrich** [comment]
then in one it is commented out so we shoild do that in both

**Comment by max-leuthaeuser** [comment]
Ok, fixed that with a22b43b.

**Comment by cdietrich** [comment]
looks better now. am still not sure if/how we can avoid the vimeo cookie.
do we have means to use a own simple videoplayer in times of html5 ?

**Comment by cdietrich** [comment]
do i see it correct; besides vimeo we dont have cookies anymore so we wont even need the consent plugin?

**Comment by max-leuthaeuser** [comment]
I will remove the iframes embedding the vimeo videos with the next commit. Instead the html video element will be utilized showing those video (270p, 360p, and 720p - served locally).
After that I will investigate all remaining cookies.

**Comment by cdietrich** [comment]
nice :+1: was playing around with the video this afternoon but i missed the `embedded-video` css class so the video had the wrong size

**Pull request merged**